app/controllers/posts_controller.rb

```diff
@@ -14,11 +15,38 @@ def create
     end
   end
 
+  def edit
+    @post = current_user.posts.find(params[:id])
+  end
+
+  def update
+    @post = current_user.posts.find(params[:id])
+
+    if @post.update(post_params)
+      flash[:notice] = "You've successfully updated your post!"
+      redirect_to @post.group
+    else
+      flash[:alert] = "Body cannot be blank"
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.